virtool/subtractions/db.py

```diff
@@ -25,6 +25,65 @@
     "has_file"
 ]
 
+FILES = [
+    "subtraction.fa.gz",
+    "subtraction.1.bt2",
+    "subtraction.2.bt2",
+    "subtraction.3.bt2",
+    "subtraction.4.bt2",
+    "subtraction.rev.1.bt2",
+    "subtraction.rev.2.bt2"
+]
+
+
+class AddSubtractionFilesTask(virtool.tasks.task.Task):
+
+    def __init__(self, app, task_id):
+        super().__init__(app, task_id)
+
+        self.steps = [
+            self.rename_index_files,
+            self.add_files_field,
+        ]
+
+    async def rename_index_files(self):
+        settings = self.app["settings"]
+
+        async for subtraction in self.db.subtraction.find({"deleted": False}):
+            path = virtool.subtractions.utils.join_subtraction_path(settings, subtraction["_id"])
+            for file in os.listdir(path):
+                if file.endswith(".bt2"):
+                    file_path = os.path.join(path, file)
+                    os.rename(file_path, file_path.replace("reference", "subtraction"))
```

Comment:
This should run in a thread in case the filesystem is slow. Modifying the file system synchronously will block the whole application.

I suggest extracting lines 54-57 to a separate function and use app["run_in_thread"] to execute.